### DISCUSSION TITLE
不同的namepath可以有相同的name吗

### DISCUSSION THREAD

**Discussion by Y-HR** [ant-design/ant-design#51482] [Q&A] [answered]
例如［"user1","userName"］、［"user2","userName"］

**Comment by petercat-assistant** [comment]
在编程中，通常情况下，不同的路径（namepath）可以拥有相同的名称（name）。这意味着在不同的上下文或命名空间中，可以使用相同的名称而不会产生冲突。例如，在不同的对象或模块中，属性或方法可以有相同的名称，只要它们的路径不同。

在Ant Design中，组件的属性名称通常是唯一的，以避免在同一上下文中出现冲突。然而，在不同的组件或模块中，属性名称可以相同，因为它们属于不同的命名空间。

如果你在使用Ant Design时遇到具体的代码问题或需要进一步的帮助，请随时告诉我！

如需进一步帮助，请在评论中描述您的问题并@petercat-assistant与我开始对话。

**Comment by crazyair** [comment]
可以，但是值会同步

**Reply by Wxh16144** [reply]
为啥会同步，不符合自觉啊

**Comment by crazyair** [comment, marked as answer]
`［"user1","userName"］、［"user2","userName"］` 这是2个 name，不一样
